### PR TITLE
ColorPickerGridCell: High-Contrast support for dark colors

### PR DESCRIPTION
**Pull request by kysedate**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes VSO item #2071482
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Adding logic to determine if a color in the color picker is 'dark'. If it is, add a white border around the icon in high contrast mode.

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/SwatchColorPicker/ColorPickerGridCell.base.tsx [hunk 1]
```diff
@@ -101,6 +102,16 @@ export class ColorPickerGridCellBase extends React.Component<IColorPickerGridCel
...
   }
 
+  /**
+* Validate if the cell's color is dark enough to apply the dark cell high contrast style
+* @param inputColor - The color of the current cell
+* @returns - Whether the cell's color is dark or not.
+*/
+  private _isDarkCell(inputColor: string | undefined): boolean {
+    let color = inputColor ? getColorFromString(inputColor) : undefined;
+    return (color && (color.r < 120 && color.g < 120 && color.b < 120)) || false;
```
Comment (jspurlin) [inline packages/office-ui-fabric-react/src/components/SwatchColorPicker/ColorPickerGridCell.base.tsx]: Let's run this by designers to make sure everyone is on the same page around what constitutes "dark".  FYI, @betrue-final-final

File: packages/office-ui-fabric-react/src/components/SwatchColorPicker/ColorPickerGridCell.styles.ts [hunk 2]
```diff
@@ -54,6 +55,20 @@ export const getStyles = (props: IColorPickerGridCellStyleProps): IColorPickerGr
           ':active $svg': getSvgSelectorStyles(ACTIVE_BORDER_COLOR, false),
         }
       },
+      isDark && {
+        selectors: {
+          [HighContrastSelector]: {
```
Comment (jspurlin) [inline packages/office-ui-fabric-react/src/components/SwatchColorPicker/ColorPickerGridCell.styles.ts]: Is there a reason why this is only added in high contrast?

File: packages/office-ui-fabric-react/src/components/SwatchColorPicker/ColorPickerGridCell.base.tsx (hunk 1 above)
Comment (jspurlin) [reply packages/office-ui-fabric-react/src/components/SwatchColorPicker/ColorPickerGridCell.base.tsx]: @betrue-final-final we need to make a call on when we are going to add a white border to the dark circle(s)... Should it be only for the black circle or if the color is darker than a specific hue? ... and if we go the hue route, should we update the isWhite logic to be isLight?

Comment (betrue-final-final) [reply packages/office-ui-fabric-react/src/components/SwatchColorPicker/ColorPickerGridCell.base.tsx]: I'm going to explore looking at the white border. We should not change swatches to be more visible but remove any accuracy.

**Pull request closed without merging**